music/demos/piano_genie_conditioned.ts

```diff
@@ -0,0 +1,244 @@
+/**
+ * @license
+ * Copyright 2019 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as Tone from 'tone';
+import * as mm from '../src/index';
+import { CHECKPOINTS_DIR } from './common';
+
+const GENIE_DIR = `${CHECKPOINTS_DIR}/piano_genie/model`;
+const GENIE_CHECKPOINTS: {[key:string]: {[key:string]: string}} = {
+  // Models conditioned on performance timing features
+  dt_only: {
```

Comment:
please use less abbreviations and more words on these names, since they're getting shown in the select box, and not everybody knows what they stand for